packages/app/src/cli/utilities/extensions/cli.ts

```diff
@@ -68,3 +84,56 @@ export async function nodeExtensionsCLIPath(): Promise<string> {
     return executablePath
   }
 }
+
+/**
+ * This method provides a Writable wraper which parses the incoming chunks as GoLog from Json and writes them back to
+ * the Writable given as parameter. If Parsing to Json fails it wirtes the chunk as it is and thows an error.
+ * @param output {Writable} A Writable which will be wrapped and in which the parsed chunks will be written.
+ * @returns {Writable} A Writable wrapping the parameter.
+ */
+export function goLogWritable(output: Writable): Writable {
+  return new Writable({
+    write(chunk, _encoding, next) {
+      const lines = outputKit.stripAnsiEraseCursorEscapeCharacters(chunk.toString('ascii')).split(/###LOG_END###/)
+      for (const line of lines) {
+        if (line) {
+          try {
+            const log = JSON.parse(line) as GoLog
+            output.write(parseGoLogMessage(log))
+            // eslint-disable-next-line no-catch-all/no-catch-all
+          } catch (err) {
+            output.write(line)
+          }
+        }
+      }
+      next()
+    },
+  })
+}
+
+function parseGoLogMessage(log: GoLog): string {
+  if (!log.level || !log.status || !log.payload.message) {
+    throw new Error(`Invalid log: ${log}`)
+  }
+  if (log.extensionName) {
+    return `${log.extensionName} ${styleWorkflowStep(log.workflowStep)} ${log.status}: ${styleContent(
+      log.payload.message,
+      log.status,
+    )}`
+  }
+  return `\x1b[94m[${log.workflowStep}]\x1b[0m ${log.status}: ${styleContent(log.payload.message, log.status)}`
```

Comment:
what are these characters? If you want to add color/text styles we have methods for that (search for `output.content` and `output.token`)